src/components/DetailsMetrics/DetailsMetrics.scss

```diff
@@ -0,0 +1,189 @@
+@import '~igz-controls/scss/shadows';
+@import '~igz-controls/scss/colors';
+
+.metrics {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 16px;
+  height: fit-content;
+
+  &__card-tmp {
+    z-index: 1;
+    top: 0;
+    position: sticky !important;
```

Comment:
we should never use `!important`. Only if it's not possible to override the style

Reply:
the property position form card class overwrite the _card-tmp. i removing the !important, will be sovle with css selectors